docs/content/reference/exploiters/_index.md

```diff
@@ -12,5 +12,3 @@ tags = ["reference", "exploit"]
 The Infection Monkey uses various remote code execution (RCE) exploiters. To our best knowledge, most of these pose no risk to performance or services on victim machines. This documentation serves as a quick introduction to the exploiters currently implemented and the vulnerabilities they use:
 
 {{% children %}}
-
-You can check out the exploiters' implementation yourself [in the Infection Monkey's GitHub repository](https://github.com/guardicore/monkey/tree/develop/monkey/infection_monkey/exploit).
```

Comment:
Do we want to remove this link? I don't have an opinion on this, I just wanted to point out that it was removed.

Reply:
From the commit:
> Hadoop is not a part of this so this is already outdated.
The rest of the exploiters will also soon be moved out of
here when we convert them to plugins.

Reply:
We'll have to have a definitive list where a user could go to to see the source code of the plugins/exploiters. I'd rather keep it and extend it with a link to Hadoop